src/org/ods/usecase/LeVADocumentUseCase.groovy

```diff
@@ -1021,8 +1021,6 @@ class LeVADocumentUseCase extends DocGenUseCase {
         }
         def sectionsNotDone = this.getSectionsNotDone(sections)
 
-        def pods = this.os.getPodDataForComponent(repo.id)
-
         def data_ = [
```

Comment:
you create in the `data` object now an openshift structure. that needs to be reflected in the lines  1030ff